site/pages/create-consequence-services/[disruptionId]/[consequenceIndex].page.tsx

```diff
@@ -64,7 +64,7 @@ const filterConfig = {
     matchFrom: "any" as const,
 };
 
-const getStops = async (
+export const getStops = async (
```

Comment:
Should we move this to utils if it's being used in multiple places?

Reply:
moved